docs/api/vulnerability-info.md

```diff
@@ -0,0 +1,114 @@
+
+A suggested strategy for vulnerability data partitioning is to have two files, `base.json` and `updates.json`.
+The `base.json` file is updated periodically (for example once a month), and contains all known vulnerabilities at the time the file is regenerated.
+The `updates.json` file should any new advisories published since `base.json` was last regenerated.
+This will allow clients to download the large `base.json` infrequently, while the frequently changing `updates.json` file is always relatively small.
+
+NuGet client does not provide a way for one file to update or remove entries from another file, so if any modifications are needed to any advisory in `base.json`, then `base.json` can be regenerated and all entries in `updates.json` merged.
+
+## Vulnerability index
+
+The vulnerability index is a JSON array of objects with the following properties:
+
+Name     | Type     | Required | Notes
+-------- | ---------| -------- | -----
+@name    | string   | yes      | A short name for the file, used as a cache key.
+@id      | string   | yes      | The full (absolute) URL to a vulnerability data file.
+@updated | DateTime | yes      | An ISO 8601 string representing the date and time the file was last updated, ideally with the UTC timezone.
+comment  | string   | no       | A optional descriptive string.
+
+The following restrictions apply:
+
+* The index must be an array of objects with between 1 and 16 items.
+* `@name` must be unique within the index, must be between 1 and 32 characters long, and can only use the characters `A` to `Z`, `a` to `z`, `0` to `9`, `-`, or `_`.
+* `@id` must be an absolute URL, not a relative URL.
+
+## Vulnerability data
+
+Vulnerability data files are a JSON object used as a dictionary.
+Property keys are package identifiers and values are an array of the following object with the following properties:
+
+Name     | Type   | Required | Notes
+-------- | -------| -------- | -----
+severity | number | yes      | 0 means low, 1 means medium, 2 means high, 3 means critical.
+url      | string | yes      | URL where users can get more information about the vulnerability.
+versions | string | yes      | The version range that is vulnerable, using [NuGet's version range syntax](../concepts/package-versioning.md#version-ranges).
+
+Package identifiers (the root object's keys) should be lowercase with [`String.ToLowerInvariant`](/dotnet/api/system.string.tolowerinvariant).
+
+## Samples
+
+Here is a sample of a vulnerability index:
+
+```json
+[
+    {
+        "@name": "base",
+        "@id": "https://nuget.contoso.com/v3/vulnerabilities/vulnerability.base.json",
+        "@updated": "2023-06-01T06:14:58.4159909Z",
+        "comment": "The base data for vulnerability update periodically"
+    },
+    {
+        "@name": "update",
+        "@id": "https://nuget.contoso.com/v3/vulnerabilities/vulnerability.update.json",
+        "@updated": "2023-06-01T06:14:58.4159909Z",
```

Comment:
Maybe provide a later timestamp to better hint at what we plan on doing in the northstar?